runtime/jit_vm/ctsupport.cpp

```diff
@@ -720,4 +720,64 @@ jitGetDeclaringClassOfROMField(J9VMThread *vmStruct, J9Class *clazz, J9ROMFieldS
 	return currentClass;
 }
 
+void
+allMethodFromSignatureEndDo(J9MethodFromSignatureWalkState *state)
+{
+	J9JavaVM *vm = state->classLoaderWalkState.vm;
+	vm->internalVMFunctions->allClassLoadersEndDo(&state->classLoaderWalkState);
+}
+
+J9Method*
+allMethodFromSignatureNextDo(J9MethodFromSignatureWalkState *state)
+{
+	UDATA result = 0;
+
+	J9JavaVM *vm = state->classLoaderWalkState.vm;
+	J9ClassLoader *classLoader = vm->internalVMFunctions->allClassLoadersNextDo(&state->classLoaderWalkState);
+	while (NULL != classLoader) {
+		J9Class *clazz = vm->internalVMFunctions->internalFindClassUTF8(state->vmThread, reinterpret_cast<U_8 *>(state->className), state->classNameLength, classLoader, J9_FINDCLASS_FLAG_EXISTING_ONLY);
+		if (NULL != clazz) {
+			result = vm->internalVMFunctions->javaLookupMethod(state->vmThread, clazz, reinterpret_cast<J9ROMNameAndSignature *>(&state->nameAndSig), NULL, J9_LOOK_JNI | J9_LOOK_NO_JAVA);
+			if (0 != result) {
+				break;
+			}
+		}
+
+		classLoader = vm->internalVMFunctions->allClassLoadersNextDo(&state->classLoaderWalkState);
+	}
+
+	return reinterpret_cast<J9Method*>(result);
+}
+
+J9Method*
+allMethodFromSignatureStartDo(J9MethodFromSignatureWalkState *state, J9JavaVM *vm, UDATA flags, char *className, U_32 classNameLength, const char *methodName, U_32 methodNameLength, const char *methodSig, U_32 methodSigLength)
```

Comment:
I thought the plan was for `const char *className`?

Reply:
This isn't possible because we need to feed the value into `internalFindClassUTF8` which wants a `U_8 *`. See previous https://github.com/eclipse/openj9/pull/11989#discussion_r581282541 which explains this. We can make the change if we `const_cast` the `const` away. That is a bit misleading to me but I can make that change. Please advise.

Reply:
You're casting it anyway; I had understood that cast would include casting the `const` away.

Reply:
Fixed in bfae119